src/lib/browse/layers/RoadWidthsLayerControl.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import { ExternalLink, HelpButton, InteractiveLayer } from "lib/common";
+  import { Checkbox } from "lib/govuk";
+  import {
+    hoveredToggle,
+    makeColorRamp,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+  } from "lib/maplibre";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "stores";
+  import { colors, denseLineWidth } from "../colors";
+  import SequentialLegend from "./SequentialLegend.svelte";
+
+  let name = "road_widths";
+  let colorScale = colors.sequential_low_to_high;
+  let limits = [0, 5, 10, 15, 20, 100];
+
+  overwritePmtilesSource(
+    $map,
+    name,
+    `${import.meta.env.VITE_RESOURCE_BASE}/private_layers/v1/${name}.pmtiles`
+  );
+
+  overwriteLineLayer($map, {
+    id: name,
+    source: name,
+    sourceLayer: name,
+    color: makeColorRamp(["get", "average"], limits, colorScale),
+    // TODO Try showing the actual width, in meters and not pixels
```

Comment:
In other words, once you're zoomed in, you see "actual" carriageways as polygons, not as thickened lines that're always the same pixel-size at any zoom. I briefly played around with this:
```
-    width: [
-      "interpolate",
-      ["exponential", 2],
-      ["zoom"],
-      10,
-      ["*", ["get", "average"], ["^", 2, -6]],
-      24,
-      ["*", ["get", "average"], ["^", 2, 8]],
-    ],
```

but it's going to take more work. It's useless (and slow) at low zoom, and "zoom" can only be used in top-level interpolate/step expressions. I think we might have to use two separate line layers glued to the same source, with different min/max zooms. Leaving as future work

Reply:
That's a super cool idea big fan